src/test/java/com/redhat/cloud/notifications/routers/BundleServiceTest.java

```diff
@@ -50,42 +51,28 @@ void testAdd() {
                 .then()
                 .statusCode(200);
 
+        updateBundle(bundleId, 200);
+
         deleteBundleById(bundleId);
 
         when()
                 .get("/internal/bundles/" + bundleId)
                 .then()
                 .statusCode(404);
+
+        updateBundle(UUID.randomUUID().toString(), 404);
     }
 
     @Test
     void testAddTwoBundles() {
-        JsonObject b1 = null;
-        JsonObject b2 = null;
-        try {
-            b1 = createBundle("b1", 200);
-            b2 = createBundle("b2", 200);
-        } finally {
-            if (b1 != null) {
-                deleteBundleById(b1.getString("id"));
-            }
-            if (b2 != null) {
-                deleteBundleById(b2.getString("id"));
-            }
-        }
+        createBundle("b1", 200);
+        createBundle("b2", 200);
     }
 
     @Test
     void testNoAddSameBundleTwice() {
-        JsonObject b1 = null;
-        try {
-            b1 = createBundle("b1", 200);
-            createBundle("b1", 500);
-        } finally {
-            if (b1 != null) {
-                deleteBundleById(b1.getString("id"));
-            }
-        }
```

Comment:
We no longer need to delete database records after tests because the database is cleaned after each test.